apps/server/src/modules/pseudonym/repo/pseudonyms.repo.spec.ts

```diff
@@ -169,4 +205,35 @@ describe('PseudonymRepo', () => {
 			});
 		});
 	});
+
+	describe('deletePseudonymsByUserId', () => {
+		describe('when pseudonyms are existing', () => {
+			const setup = async () => {
+				const user = userFactory.buildWithId();
+				const pseudonym1: PseudonymEntity = pseudonymEntityFactory.buildWithId({ userId: user.id });
+				const pseudonym2: PseudonymEntity = pseudonymEntityFactory.buildWithId({ userId: user.id });
+
+				await em.persistAndFlush([pseudonym1, pseudonym2]);
+
+				return {
+					user,
+				};
+			};
+
+			it('should delete all pseudonyms for userId', async () => {
+				const { user } = await setup();
+
+				const result: number = await repo.deletePseudonymsByUserId(user.id);
+
+				expect(result).toEqual(2);
+			});
+		});
```

Comment:
Could you please add some more (1-2) pseudonyms objects not related to the the given userId to verify it's for sure deleting just the pseudonyms related to the given userId and not to any other userId?